pipeline/Dockerfile.dev

```diff
@@ -20,7 +20,7 @@ WORKDIR /mnt/project
 
 # Build dependencies.
 COPY --chown=opam:opam pipeline/pipeline.opam pipeline.opam
-RUN opam install -y --deps-only -t .
+RUN opam install -y --deps-only .
```

Comment:
installing with the `-t` flag makes it longer without a good motive. In the prod Dockerfile we already don't use it